README.md

```diff
@@ -694,6 +694,38 @@ class MyThemeableWidget extends ThemeableMixin(WidgetBase)<MyThemeableWidgetProp
 
 The theme can be applied to individual widgets or to a project and property passed down to its children.
 
+##### Injecting a theme
+
+The theming system supports injecting a theme that is configured outside of the usual property passing down the widget tree. 
```

Comment:
"The theming system supports injecting a theme that is configured outside of the usual property passing down the widget tree. " perhaps -> "The theming system supports injecting a theme that is configured externally to the usual mechanism of passing properties down the widget tree. "?